app/views/static/ems_cloud/arbitration_profile_edit.html.haml

```diff
@@ -30,22 +30,21 @@
     %label.col-md-2.control-label{"for" => "authentication_id"}
       = _('Key Pair')
     .col-md-8
-      %select{"ng-model"    => "arbitrationProfileModel.authentication_id",
+      -#%select{"ng-model"    => "arbitrationProfileModel.authentication_id",
+      -#        "name"        => "authentication_id",
+      -#        'ng-options'  => 'authentication.id as authentication.name for authentication in arbitrationProfileModel.authentications track by authentication.id',
+      -#        'pf-select'   => true}
+      -#  %option{"value" => ""}
+      -#    = "<#{_('Choose')}>"
+
+      %select{"ng-model"    => "$scope._authentication",
+              "ng-init"     => "$scope._authentication = _.find($scope.arbitrationProfileModel.authentications, {id: '1000000000054'})",
```

Comment:
This is wrong, don't put it in ng-init, put it in the controller, *after* you've actually retrieved those authentications..